examples/Building-and-deploying-multi-stage-RecSys/01-Building-Recommender-Systems-with-Merlin.ipynb

```diff
@@ -106,9 +106,19 @@
     "### Import required libraries and functions"
```

Comment:
<p>Where is BASE_DIRE defined? Should that not crash?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/244/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Reply:
Never mind - ReviewNB had the section collapsed and therefore, I did not see it.